fews_3di/simulation.py

```diff
@@ -149,25 +149,41 @@ def run(self):
         self.simulation_id, self.simulation_url = self._create_simulation(model_id)
 
         laterals_csv = self.settings.base_dir / "input" / "lateral.csv"
-        laterals = utils.lateral_timeseries(laterals_csv, self.settings)
-        self._add_laterals(laterals)
+        if path.exists(laterals_csv):
+            laterals = utils.lateral_timeseries(laterals_csv, self.settings)
+            self._add_laterals(laterals)
+        else:
+            logger.info("No lateral timeseries found, skipping...")
 
         saved_state_id_file = self.settings.base_dir / SAVED_STATE_ID_FILENAME
         if self.settings.save_state:
             self._add_initial_state(saved_state_id_file)
             self.saved_state_id = self._prepare_initial_state()
+        else:
+            logger.info("Not using saved state, skipping...")
 
         rain_file = self.settings.base_dir / "input" / "precipitation.nc"
-        rain_raster_netcdf = utils.write_netcdf_with_time_indexes(
-            rain_file, self.settings
-        )
-        self._add_rain(rain_raster_netcdf)
+        if path.exists(rain_file):
+            rain_raster_netcdf = utils.write_netcdf_with_time_indexes(
+                rain_file, self.settings
+            )
+            self._add_rain(rain_raster_netcdf)
+        else:
+            logger.info("No rain file found, skipping...")
 
         evaporation_file = self.settings.base_dir / "input" / "evaporation.nc"
-        evaporation_raster_netcdf = utils.write_netcdf_with_time_indexes(
-            evaporation_file, self.settings
-        )
-        self._add_evaporation(evaporation_raster_netcdf)
+        if path.exists(evaporation_file):
+            evaporation_raster_netcdf = utils.write_netcdf_with_time_indexes(
+                evaporation_file, self.settings
+            )
+            self._add_evaporation(evaporation_raster_netcdf)
+        else:
+            logger.info("No evaporation file found, skipping...")
+
+        if self.settings.process_basic_results:
```

Comment:
Is deze nieuwe setting een expliciete wens?

Het lijkt me nogal een essentieel onderdeel. Dus: niet een setting toevoegen "omdat het misschien handig is", maar alleen als het echt nodig is. Even voor de zekerheid.

----

Ah, ik zie dat het überhaupt nieuwe functionaliteit is.